### PR TITLE
Move project metadata to pyproject.toml

### PR DESCRIPTION
**Pull request by EpicWink**
See [Python packaging docs](https://packaging.python.org/en/latest/guides/writing-pyproject-toml) and [setuptools docs](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html).

Depends on staticshock/colored-traceback.py#21

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,35 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools >= 36.6"]
+requires = ["setuptools >= 61.0"]
+
+[project]
+name = "colored"
```
Comment (hartwork) [inline pyproject.toml]:
```suggestion
name = "colored-traceback"
```

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "License :: OSI Approved :: ISC License (ISCL)",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
```
Comment (hartwork) [inline pyproject.toml]: If the mission is to translate without changing then 2.7 + 3.3 would be that translation. Else, if the mission is to also get onto supported Python version ground then 3.8 would be the minimum, [3.7 is already end of life](https://endoflife.date/python). What do you think?

File: pyproject.toml [hunk 3]
```diff
@@ -1,3 +1,35 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools >= 36.6"]
+requires = ["setuptools >= 61.0"]
```
Comment (hartwork) [inline pyproject.toml:3]: I'd be curious about which part needs this bump.

Comment (EpicWink) [reply pyproject.toml:3]: Setuptools v61 is the first version which supports project metadata in project.toml

File: pyproject.toml (hunk 2 above)
Comment (EpicWink) [reply pyproject.toml]: Setuptools v61 only supports Python 3.7+, but I think here is not the place to worry about that: I'll leave the classifiers unchanged.

**Comment by EpicWink** [comment]
See staticshock/colored-traceback.py#23

**Comment by hartwork** [comment]
@EpicWink why did you close the pull request?

**Comment by hartwork** [comment]
@EpicWink nevermind, I missed that https://github.com/staticshock/colored-traceback.py is upstream

**Pull request closed without merging**